sdk/playwrighttesting/create-microsoft-playwright-testing/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2024-11-15)
+
+This release includes updates to Engineering Systems and the migration of the package to ECMAScript modules and Vitest.
```

Comment:
Can we say something other than Engineering systems that will be relevant for the customers?